src/index.js

```diff
@@ -2,57 +2,67 @@
 import parse from 'postcss-value-parser'
 import camelizeStyleName from 'camelize'
 import transforms from './transforms/index'
+import devPropertiesWithoutUnitsRegExp from './devPropertiesWithoutUnitsRegExp'
 import TokenStream from './TokenStream'
 
 // Note if this is wrong, you'll need to change tokenTypes.js too
-const numberOrLengthRe = /^([+-]?(?:\d*\.)?\d+(?:[Ee][+-]?\d+)?)(?:px)?$/i
+const numberOrLengthRe = /^([+-]?(?:\d*\.)?\d+(?:[Ee][+-]?\d+)?)(?:px)?$/
```

Comment:
Hmm.. shouldn't this havew the `/i` to match the unit if it is written in uppercase?

Reply:
...or can you even write the values uppercase in React Native?

Reply:
Good catch! I only saw the `[Ee]`. Updated this now

Reply:
> ...or can you even write the values uppercase in React Native?
 
The only units RN ever takes is `%`, and `deg` or `rad` in transforms. Just updated these to handle this